app/localization.js

```diff
@@ -21,25 +21,21 @@ export function Localizer() {
 Localizer.prototype = {
     // Configure suitable language based on user preferences
     setup: function (supportedLanguages) {
-        var userLanguages;
-
         this.language = 'en'; // Default: US English
 
         /*
          * Navigator.languages only available in Chrome (32+) and FireFox (32+)
          * Fall back to navigator.language for other browsers
          */
-        if (typeof window.navigator.languages == 'object') {
-            userLanguages = window.navigator.languages;
-        } else {
-            userLanguages = [navigator.language || navigator.userLanguage];
-        }
+        const userLanguages = (typeof window.navigator.languages === 'object')
+            ? window.navigator.languages
+            : [navigator.language || navigator.userLanguage];
 
-        for (var i = 0;i < userLanguages.length;i++) {
-            var userLang = userLanguages[i];
-            userLang = userLang.toLowerCase();
-            userLang = userLang.replace("_", "-");
-            userLang = userLang.split("-");
+        for (let i = 0;i < userLanguages.length;i++) {
+            const userLang = userLanguages[i]
+                .toLowerCase()
+                .replace("_", "-")
+                .split("-");
```

Comment:
Perhaps we should move this to a helper function instead? Avoids this churn, and makes the `const` declaration more clear.